tests/integration/test_security_report.py

```diff
@@ -46,7 +46,7 @@ def test_uses_origins(mini_sentry, relay, json_fixture_provider, allowed_origins
     )
 
     if should_be_allowed:
-        mini_sentry.captured_events.get(timeout=1).get_event()
+        mini_sentry.captured_events.get(timeout=10).get_event()
```

Comment:
Note: This is necessary because the non-processing relay now does normalization, which includes UA parsing, which apparently has slow initialization.